File: 36-Riyad/src/lab1/Cricket.java
package lab1;

public class Cricket extends Game{
	
	String teamName;
	String gametype;
	String score;
	String bestBatter;
	String bestBowler;
	
	public Cricket(String teamName, String gameType,String score, String bestBatter, String bestBowler) {
		this.teamName = teamName;
		this.gametype = gameType;
		this.score = score;
		this.bestBatter = bestBatter;
		this.bestBowler = bestBowler;
		
	}

	@Override
	void teamInfo() {
		System.out.println(teamName);
		System.out.println(gametype);
		System.out.println("== == ==");
		
	}

	@Override
	void score() {
		System.out.println(score);
		System.out.println("== == ==");
		
	}

	@Override
	void bestPlayer() {
		System.out.println("Best Batter: "+bestBatter);
		System.out.println("Best Bowler: "+bestBowler);
		
	}

}

File: 38-tahmid/Task2/ScoreBoard/Cricket.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ScoreBoard
{
    class Cricket :ScoreBoard
    {
        String team1, team2, score, bestBatsman,bestBowler, gameType,interval = "== == ==";
       public Cricket(String team1, String team2, String score, String bestBatsman,String bestBowler,String gameType)
        {
            this.team1 = team1;
            this.team2 = team2;
            this.score = score;
            this.bestBatsman = bestBatsman;
            this.bestBowler = bestBowler;
        }
        public void display()
        {
            System.Console.WriteLine(team1 + " vs " + team2);
            System.Console.WriteLine(interval);
            System.Console.WriteLine(score);
            System.Console.WriteLine(interval);
            System.Console.WriteLine("Best Batter:" + bestBatsman);
            System.Console.WriteLine("Best Bowler:" + bestBowler);


        }

        public void disPlay()
        {
            System.Console.WriteLine(team1 + " vs " + team2);
            System.Console.WriteLine(interval);
            System.Console.WriteLine(score);
            System.Console.WriteLine(interval);
            System.Console.WriteLine("Best Batter:" + bestBatsman);
            System.Console.WriteLine("Best Bowler:" + bestBowler);
        }
    }
}
